magda-search-api/src/main/scala/au/csiro/data61/magda/search/elasticsearch/ElasticSearchQueryer.scala

```diff
@@ -321,12 +321,12 @@ class ElasticSearchQueryer(indices: Indices = DefaultIndices)(
     val clauses: Seq[Traversable[QueryDefinition]] = Seq(
       query.freeText flatMap { inputText =>
         val text = if (inputText.trim.length == 0) "*" else inputText
-        val queryString = new SimpleStringQueryDefinition(text).defaultOperator(operator).quoteFieldSuffix("keyword")
+        val queryString = new SimpleStringQueryDefinition(text).defaultOperator(operator).quoteFieldSuffix(".quote")
 
         // For some reason to make english analysis work properly you need to specifically hit the english fields.
         def foldFieldsEnglish(query: SimpleStringQueryDefinition, fields: Seq[Any]) = fields.foldRight(query) {
```

Comment:
Forgot to do this in mind - you can get rid of this function, it's now a copy paste of foldFields